packages/siop-oid4vp/lib/helpers/Revocation.ts

```diff
@@ -10,25 +16,27 @@ export const verifyRevocation = async (
   if (!vpToken) {
     throw new Error(`VP token not provided`)
   }
-  if (!revocationVerificationCallback) {
-    throw new Error(`Revocation callback not provided`)
-  }
+  if (isWrappedW3CVerifiablePresentation(vpToken) || isWrappedSdJwtVerifiablePresentation(vpToken)) { // Only W3C support for now
```

Comment:
Please invert that and create a debug line stating it is not supported for non-w3c and return immediately, instead of creating a big if